R/get-path-from-object.R

```diff
@@ -20,7 +33,13 @@ get_model_path <- function(.bbi_object, .check_exists = TRUE) {
 #' @rdname get_path_from_object
 #' @export
 get_model_path.bbi_nonmem_model <- function(.bbi_object, .check_exists = TRUE) {
-  find_nonmem_model_file_path(.bbi_object[[ABS_MOD_PATH]], .check_exists)
+  get_model_path_bbi(.bbi_object, .check_exists)
```

Comment:
The methods for `bbi_nonmem_model` and `bbi_nonmem_summary` are identical for each of the `get_*` functions. Another approach to this problem could therefore be to create an abstract (parent) class from which both sub-classes inherit, and then define the method for the abstract class rather than writing the same method twice. The methods are all small enough that it's not adding a lot of code to write them twice, but it does complicate keeping the methods in sync if and when they change.

Reply:
Per discussion offline, I think this is the right idea, but we are going to wait on it for now. The code as is works and will not be difficult to refactor whenever we cross this bridge. Adding a parent class seems like a good idea moving forward, but it will require some forward-thinking design that isn't necessary for this PR.